internal/paascharm/gostructs.go

```diff
@@ -0,0 +1,142 @@
+package paascharm
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"unicode"
+)
+
+const CommonPrefix = "APP_"
```

Comment:
Is this always `APP_`? Shouldn't the user be able to override it, for example with a command line parameter to the generator?

Reply:
The environment variables are prefixed in this way, the reason is namespacing. So a CLI arg here wouldn't help since it is tied to the environment variable. Do you think the prefix is reasonable?